src/constants.js

```diff
@@ -9,8 +9,15 @@ module.exports.ExitCode = {
 
 module.exports.HttpCode = {
   OK: 200,
+  NO_CONTENT: 204,
+  BAD_REQUEST: 400,
   NOT_FOUND: 404,
   INTERNAL_SERVER_ERROR: 500,
   FORBIDDEN: 403,
   UNAUTHORIZED: 401,
+  CREATED: 201,
 };
+
+module.exports.MAX_ID_LENGTH = 6;
+
+module.exports.API_PREFIX = `/api`;
```

Comment:
Я за то - что бы был один экспорт


```suggestion
const MAX_ID_LENGTH = 6;
const API_PREFIX = `/api`;

module.exports = {
   MAX_ID_LENGTH,
   API_PREFIX
}
```